environments/environments.py

```diff
@@ -84,15 +84,7 @@ def _evolution(self, total_time=10, delta_time=0.1):
         y0 = np.array([np.array(self.q), np.array(self.p)]).reshape(-1)
         self._rollout = solve_ivp(self._dynamics, t_span, y0, t_eval=t_eval).y
 
-    def sample_random_rollouts(self,
-                               number_of_frames=100,
-                               delta_time=0.1,
-                               number_of_rollouts=16,
-                               img_size=32,
-                               color=True,
-                               noise_std=0.,
-                               base_radius=1.3,
-                               seed=None):
+    def sample_random_rollouts(self, number_of_frames=100, delta_time=0.1, number_of_rollouts=16, img_size=32, color=True, noisy_data=False, noise_std=0.1, radius_lb=1.3, radius_ub=2.3, seed=None):
```

Comment:
I think its better if we use autopep, easier to read all the arguments.

Also I removed the noisy data flag, if you dont want noisy data just set the std dev to 0. The idea was to not have redundant arguments